### ISSUE TITLE
A few server issues to report after some time on version 3.4.4 Mythic

### ISSUE THREAD

**Issue by Wafool** [Cazadorsniper/PO3#1592]
Hello! We love Project Ozone 3 and would like to report some issues we've run into while running this pack on one of our servers. I run a community with a few servers dedicated to expert packs. We hosted kappa on 3.3.61 for a while, and it was great. Well, aside from translocators. those things had to be removed. But anyway, We have our own machine and run forge only. Why use plugins when there's mods for all the same things? We have installed dupefix, spark, chunk pregenerator, kubejs (for only an automatic rank promotion script) and lag goggles (v4.11.92 to avoid the compatibility issue with enderio). I have tested these issues without these mods on a fresh install, and they do still persist.

NetherEx doesn't seem to be generating it's biome's properly. It look's like it tried to work for a bit, at least near world spawn coordinates. There are "microbiomes" from the netherex mod there. But aside from the few regions around 0,0 there are no netherex biomes at all. It's all hell biome as far as the eye can see with some nether fortress biomes here and there. On top of that, these hell biomes do not seem to spawn mobs. At all. 
![PRisc0U](https://user-images.githubusercontent.com/64870736/142778166-cba61281-22a9-4d4a-853a-e94747de7194.jpg)
That's an image of the contrast between a biome that normally spawns mobs (nether fortress) and the hell biome which seems to be broken and won't spawn mobs.

Which leads me to another thing. The mob spawns are absolutely insane. If left dark for only a a few seconds, a chunk will be full shoulder to shoulder of mobs. We do have about 6-10 players online at all times, and with the pack being a skyblock mostly, maybe there's just not many places for mobs to spawn. Spark profilers shows very high usage from find chunks for spawn. Seems like there's a lot of checks for mob spawning going on. I can probably remedy this with another mod like InControl, but I'm curious if another mod is increasing normal spawn rates. Like Esm?

Also, stonelings from quark are an issue. We had two crashes from them. The first of which, a player tried to capture one in an enderio soul vial. This crash ended up causing a world corruption. We had a ticking world crash loop that ended up needing a rollback to remedy
[crash-2021-11-20_03.55.58-server.txt](https://github.com/Cazadorsniper/PO3/files/7577082/crash-2021-11-20_03.55.58-server.txt)
I have since raise the max tick time to a pretty absurd amount. But, there was another crash that happened and the player said he was just mining close by one this second time.
[crash-2021-11-20_18.32.13-server.txt](https://github.com/Cazadorsniper/PO3/files/7577087/crash-2021-11-20_18.32.13-server.txt)
I have removed stonelings in Quark's configs.

Anyway, just hoping to share some info that may help with bugfixes in future updates. If there's any further info you'd like, for example logs or more crash reports or something, just let me know. Thanks for all the work you guys do in making and maintaining such a great pack.

**Comment by Wafool** [comment]
Oh, also would like to add that the netherex issue probably does need some kind of fix. The Ghast Queen tears are needed for progression in the Philo Stone recipe. Without netherex biomes generating, My players have no way of summoning the ghast queen. I did test it in a single player world and the netherex biomes generated just fine. Maybe there's just something missing from the server files that is included in the single player files.

**Comment by Cazadorsniper** [comment]
Found [#267](https://github.com/LogicTechCorp/NetherEx/issues/267) in the netherex github.

**Comment by Cazadorsniper** [comment]
ESM will be rolled backed to a previous version and nether ex might have to be rolled back as well.

**Comment by Wafool** [comment]
I see that report you linked. [#267](https://github.com/LogicTechCorp/NetherEx/issues/267). I do want to mention that we use forge and only forge. No Mohist, No Sponge, No Magma. Our server is running with only the server files provided and the 5 mods I've added for profiling and server statistics mostly. I did test without these added mods on a fresh install with only the server files and the NetherEx issue persisted.
